site/src/lib/api/blocks/r2.ts

```diff
@@ -143,9 +143,22 @@ const uploadBlockFilesToR2 = (
         `${remoteStoragePrefix}/${thingName}`,
       );
     }
-    const fileContents = fs.readFileSync(pathToThing).toString();
+    const fileContents = fs.readFileSync(pathToThing);
```

Comment:
Fixing this existing bug which was revealed by using pngs – they shouldn't be converted to strings when uploading